### PR TITLE
Privatise WebRequest dictionaries

### PR DESCRIPTION
**Pull request by smoogipoo**
We need to have the ability to null-check values placed in these dictionaries.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,31 +1,32 @@
 {
     "version": "0.2.0",
     "configurations": [{
-        "name": "VisualTests (Debug)",
-        "windows": {
-            "type": "clr"
+            "name": "VisualTests (Debug)",
```
Comment (peppy) [inline .vscode/launch.json]: too deep?

Comment (peppy) [reply .vscode/launch.json]: also would rather this was in its own PR really...

Comment (smoogipoo) [reply .vscode/launch.json]:
```
"myArray: [
    {
        "myObjectKey": "myObjectValue"
    }
]
```

->>>

```
"myArray: [{
        "myObjectKey": "myObjectValue"
    }
]
```

Your judgement is as good as mine, but this is vscode auto-formatting it.

Comment (smoogipoo) [reply .vscode/launch.json]: Also launch.json in osu! project is formatted the same way.

Comment (peppy) [reply .vscode/launch.json]: lame

**Pull request merged**